pyramid_zipkin/request_helper.py

```diff
@@ -123,6 +124,10 @@ def is_tracing(request):
     elif 'X-B3-Sampled' in request.headers:
         return request.headers.get('X-B3-Sampled') == '1'
     else:
+        override_func = request.registry.settings.get('zipkin.tracing_override')
+        if override_func and override_func(request):
```

Comment:
So if `override_func` returns false it agains falls back to %age? We should describe all this [in the docs/src](http://pyramid-zipkin.readthedocs.org/en/latest/configuring_zipkin.html).

We should handle any exception being thrown by the callback, the request should not fail